README_wasm_of_ocaml.md

```diff
@@ -30,7 +30,7 @@ opam install dune.3.17.0 wasm_of_ocaml-compiler
 You may want to install additional packages. For instance:
 
 ```
-opam install js_of_ocaml-ppx js_of_ocaml-lwt
+opam install js_of_ocaml js_of_ocaml-ppx js_of_ocaml-lwt
 ```
 
```

Comment:
I think it would be useful to explain how to setup a switch so that the testsuite can run.